compiler/test/suites/modules.re

```diff
@@ -126,44 +126,4 @@ describe("modules", ({test, testSkip}) => {
     "nestedModules",
     "hello from foo\nhello from bar\n[2, 3, 4]\n9\n[> 2, 3, 4]\nfalse\nfoo\n",
   );
-  test("reprovided_module", ({expect}) => {
```

Comment:
Would it make sense to change this to a snapshot test of the object file to ensure we are representing these correctly there.

Reply:
Done.